src/routes/registration.go

```diff
@@ -11,7 +12,7 @@ import (
 
 func (a *APIService) addRegistrationRoutes(e *echo.Echo) {
 	api := e.Group("/v1")
-	api.POST("/", a.peerUpdate)
+	api.POST("/peers", a.peerUpdate)
```

Comment:
I wasn't sure what to call this end point. /peers and /p2p are common in blockchains. /cluster and a bunch of other things would make sense too. I don't have a strong opinion on what we choose

Reply:
I like peers. Lets go with that